hybrisCustom/custom-ext/bonstore/bonstorecore/src/org/bonstore/core/event/CustomerEventListener.java

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
Could you please clarify why do we need this chain Interceptor-Event-Listener here
We can discuss it on Monday